include/configs/rpi.h

```diff
@@ -61,8 +61,8 @@
 /* GPIO */
 #define CONFIG_BCM2835_GPIO
 /* LCD */
-#define CONFIG_LCD_DT_SIMPLEFB
-#define CONFIG_VIDEO_BCM2835
+/*#define CONFIG_LCD_DT_SIMPLEFB
+#define CONFIG_VIDEO_BCM2835*/
```

Comment:
Remove these lines instead of commenting them

Reply:
Done